tests/integration_tests/test_wave_01_02_03.py

```diff
@@ -2,8 +2,8 @@
 from swap_meet.vendor import Vendor
 from swap_meet.item import Item
 
-@pytest.mark.skip
-@pytest.mark.integration_test
+# @pytest.mark.skip
+# @pytest.mark.integration_test
```

Comment:
We don't need to remove the integration test marker. Removing it didn't hurt anything in this case, but the tag can be used to help tell pytest what kinds of tests to run at certain times.